products/jira.md

```diff
@@ -143,6 +143,54 @@ releases:
     link:
       https://confluence.atlassian.com/jirasoftware/jira-software-8-20-x-release-notes-1086411771.html
 
+-   releaseCycle: "8.19"
+    releaseDate: 2021-08-25
+    eol: 2023-08-26
+    latest: "8.19.1"
+    latestReleaseDate: 2021-09-14
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-19-1-1086395933.html
+
+-   releaseCycle: "8.18"
+    releaseDate: 2021-07-01
```

Comment:
```suggestion
    # should be 2021-07-01, but 8.18.0 is not listed on https://www.atlassian.com/software/jira/update so auto-update does not consider this version.
    releaseDate: 2021-07-21
```

8.18.x release date will be automatically updated to 2021-07-21 based on data from https://github.com/endoflife-date/release-data/blob/main/releases/jira.json, because 8.18.0 is not listed on https://www.atlassian.com/software/jira/update. That's a minor inconvenience, I don't think we should do something about it except documenting it..